### DISCUSSION TITLE
ERROR: destination path already exists when trying to connect a FORKED GitHub repo

### DISCUSSION THREAD

**Discussion by TrinityChristiana** [nss-evening-cohort-14/evening-client-side#110] [Discuss a Topic]
When cloning a directory you might run into an error that looks like this:

```sh
fatal: destination path 'evening-client-side' already exists and is not an empty directory.
```

Don't worry. This just means that the repo you're trying to clone down shares its name with another directory in the same path you're trying to clone it in. You can solve this in three ways.

## Choose a different path
You can add the cloned directory to another folder. To do this just go into another directory and run the clone command.
```sh
cd <another-directory-location>
git clone <clone-url>
```
## Rename new directory
You can change the name of the cloned directory using this command:
```sh
git clone <clone-url> <new-directory-name>
```

So if I were to run the following example, the cloned repo will be named `new-repo-name`

```sh
git clone git@github.com:nss-evening-cohort-14/evening-client-side.git new-repo-name
```

## Rename the original directory
I put this one last because it's not ideal, but it still is an option. You can rename the conflicting directory before cloning the new one down.

```sh
mv <original-directory-name> <new-directory-name> # rename the original file
git clone <directory-to-clone> # then clone your directory
```

**Comment by drteresavasquez** [comment]
@TrinityChristiana NOICE!!!
